deploy/flux-helm-release-crd.yaml

```diff
@@ -102,6 +102,9 @@ spec:
                   description: If set, will wait until the minimum number of Pods of a Deployment
                     are in a ready state before marking the release as successful
                   type: boolean
+            skipCRDs:
+              description: If supplied will skip helm3 CRD installation
```

Comment:
```suggestion
              description: If set, will skip CRD installation for Helm v3
```

Reply:
Done

Reply:
Things seem to have disappeared?

Reply:
They did? looks right to me just moved the position of the skipCRDs key to be below `resetValues`?